app/src/client/resources.ts

```diff
@@ -47,6 +49,7 @@ export class TokenTransfer {
       Number(obj.blockNumber),
       String(obj.blockHash),
       dateFromString(obj.blockDate as string),
+      CollectionToken.fromObject(obj.token as Record<string, unknown>),
```

Comment:
i think this is optional in the api, can you hceck the homepage when you run locally cos this is used there too i think?